rel-eng/releasers.conf

```diff
@@ -18,3 +18,9 @@ releaser = tito.release.KojiReleaser
 autobuild_tags = foreman-plugins-nightly-nonscl-rhel7 foreman-plugins-nightly-rhel7
 builder = tito.builder.FetchBuilder
 builder.rpmbuild_options = --define "foremandist .fm1_17"
+
+[copr-nightly]
```

Comment:
Thinking we should rename this copr-foreman-nightly to distinguish it from plugins?